### DISCUSSION TITLE
Is there a way to turn off, opt-out of the embedded debugger tool?

### DISCUSSION THREAD

**Discussion by eballeste** [vimeo/player.js#879] [Q&A]
Apologies if this is a weird question but we've been having issues with an accessibility scanning tool we use to audit our page which is reporting multiple WCAG2.1 failures related to multiple page header `<title>` elements in a single page with invalid characters in them, specifically, the tool is registering multiple empty `<title></title>` tags as the output of the scan.

After looking into this, we noticed that the only common thing with all pages that report this issue is that they all play embedded Vimeo videos. We noticed by looking through the source that these embeds also include some type of debugger tool that is running and manipulating various  svg `<title>` elements that eventually are completely removed from the DOM. We think that the scan tool is incorrectly registering multiple svg title tags in their initial empty loaded state as page header elements. Does this make sense? I have no idea what else could be causing this. 😩😪

**Comment by rkrishnan8594** [comment]
We deployed a change that have fixed this issue for you. Could you try again?

**Reply by eballeste** [reply]
Hi! Question, what change was deployed? Could you give a little bit more info? Do you mean internally by Vimeo, I'm guessing yes because I'm not seeing any updates in this specific repo.

**Reply by rkrishnan8594** [reply]
Yes, internally.

**Reply by eballeste** [reply]
Gotcha, was the debugger completely nuked out of the embed? Will this be followed up with a way to turn the debugger tool back on-demand?

**Reply by rkrishnan8594** [reply]
No, the debug panel is still available. Press "d" when the player is focused.
